src/libs/suggest.js

```diff
@@ -24,6 +25,17 @@ export const selectItem = (selectedItem, replaceUrl = false) => {
     window.app.navigateTo(`/places/?type=${selectedItem.name}`,
       {}, { replace: replaceUrl });
   } else if (selectedItem instanceof Intention) {
+    Telemetry.add(
+      Telemetry.SUGGEST_CLICK,
+      null,
+      null,
+      {
+        useNlu: geocoderConfig.useNlu,
+        category: selectedItem.category ? selectedItem.category.name : null,
+        hasFullTextQuery: !!selectedItem.fullTextQuery,
+        hasPlace: !!selectedItem.place,
```

Comment:
Just sending the `place` and `fullTextQuery` would be more flexible for the future, and more consistent with what we do with the `category`. What do you think @amatissart ?

Reply:
I suggested to use these flags, as they are precise enough to build the metrics that we plan to follow. The raw queries are already processed on the back-end.  
Moreover, `selectedItem.place` is an object here, so it would require some logic to be validated on the server. I am ok to keep it simple for now.